Pod/Classes/APMAlertAnimation.swift

```diff
@@ -45,22 +45,23 @@ class APMAlertAnimation: NSObject, UIViewControllerAnimatedTransitioning {
                 completion: {
                     finished in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/b9b5565bf95c3bb2d086701d4baaa400ce0ebf36/Pod/Classes/APMAlertAnimation.swift#L46"></span>Closure parameters should be on the same line as opening brace.</td>
    </tr>
  </tbody>
</table>